modules/in_game_brief/functions/IGBRIEF/fn_missionConfirmedBlu.sqf

```diff
@@ -5,6 +5,6 @@ BlueBoard removeAction GVAR(canceledBlu);
 
 [-2, {hint "Blufor has signaled they are ready to begin!";}] call CBA_fnc_globalExecute;
```

Comment:
No reason to use CBA_fnc_globalExecute anymore, either remoteExec or a cba event

Reply:
Done.